src/modules/Elsa.Workflows.Runtime/Services/DefaultActivityExecutionMapper.cs

```diff
@@ -95,39 +83,35 @@ public ActivityExecutionRecord Map(ActivityExecutionContext source)
             Status = GetAggregateStatus(source),
             CompletedAt = source.CompletedAt
         };
-    }
-
-    private static LogPersistenceMode GetDefaultPersistenceMode(IDictionary<string, object> customProperties,Func<LogPersistenceMode> defaultFactory)
-    {
-        var properties = customProperties
-            .GetValueOrDefault<IDictionary<string, object?>>(LogPersistenceModeKey, () => new Dictionary<string, object?>());
-        var persistencePropertyDefault = properties!
-            .GetValueOrDefault("default", defaultFactory);
-
-        if(persistencePropertyDefault == LogPersistenceMode.Default) 
-            return defaultFactory();
-        return persistencePropertyDefault; 
-    }
-
-    private static Dictionary<string,object?> StorePropertyUsingPersistanceMode(IDictionary<string,object?> inputs
-        , IDictionary<string,object> persistenceModeConfiguration
-        , LogPersistenceMode defaultLogPersistenceMode = LogPersistenceMode.Exclude)
-    {
-        var result = new Dictionary<string, object?>();
-
-        foreach (var input in inputs)
-        {
-            var persistence = persistenceModeConfiguration.GetValueOrDefault(input.Key.Camelize(), () => defaultLogPersistenceMode);
-            if (persistence.Equals(LogPersistenceMode.Include) 
-                || (persistence.Equals(LogPersistenceMode.Default) && defaultLogPersistenceMode == LogPersistenceMode.Include)
-                )
-                result.Add(input.Key, input.Value);
-            
-        }
-
-        return result;
-    }
-
+    }
+
+    private static LogPersistenceMode GetDefaultPersistenceMode(IDictionary<string, object> customProperties,Func<LogPersistenceMode> defaultFactory)
+    {
+        var properties = customProperties.GetValueOrDefault<IDictionary<string, object?>>(LogPersistenceModeKey, () => new Dictionary<string, object?>());
+        var persistencePropertyDefault = properties!.GetValueOrDefault("default", defaultFactory);
+
+        if(persistencePropertyDefault == LogPersistenceMode.Default) 
+            return defaultFactory();
+        return persistencePropertyDefault; 
+    }
+
+    private static Dictionary<string,object?> StorePropertyUsingPersistenceMode(IDictionary<string,object?> inputs,
+        IDictionary<string,object> persistenceModeConfiguration,
+        LogPersistenceMode defaultLogPersistenceMode = LogPersistenceMode.Exclude)
```

Comment:
Should the default not be Include?

Reply:
Doesn't matter, the parameter is always supplied. I will remove the default value.